src/1_create_canonical.jl

```diff
@@ -136,8 +136,8 @@ string_cols = contains.(string.(typeof.(eachcol(output_features))), "String")
 output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String")] .= String.(output_features[:, string_cols])
 
 # Save geopackage
-GDF.write(joinpath(OUTPUT_DIR, "rrap_shared_lookup_$(today()).gpkg"), output_features; crs=GFT.EPSG(4326))
+GDF.write(joinpath(OUTPUT_DIR, "rrap_shared_lookup.gpkg"), output_features; crs=GFT.EPSG(4326)) #removed time so name for next script is always the same
```

Comment:
Ideally, we'd have a function that searches the directory for the file.
Where more than 1 match is found, find the one with the most recent date.

This way, we're sure when the file was generated/what version it corresponds to.